pwa/src/services/dateTime.tsx

```diff
@@ -19,3 +19,19 @@ export const dateTime = (language: TLanguageCodes, dateTime: string) => {
     </div>
   );
 };
+
+export const dateTimeTag = (language: TLanguageCodes, dateTime: string) => {
```

Comment:
This doesn't return a tag, so maybe the name could be more descriptive. E.g. `formatDateTime`?

And we could also add a return type to this function.